pom.xml

```diff
@@ -999,6 +997,7 @@
                 <module>java-native</module>
                 <module>jsoup</module>
                 <module>ksqldb</module>
+				<module>libraries-2</module>
```

Comment:
let's fix the formatting here and use spaces instead of tabs

Reply:
Done